.github/workflows/ci.yml

```diff
@@ -1,32 +1,84 @@
 name: CI
 
-on: [push, pull_request]
+on: 
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  V_HOST: localhost
+  V_PORT: 5433
+  V_USER: dbadmin
+  V_DATABASE: VMart
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    services:
-      postgres:
-        image: postgres:11
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: postgres
-          POSTGRES_DB: ci_db_test
-        ports:
-          - 5432:5432
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
     strategy:
       matrix:
-        node: ['8', '10', '12', '14', '16', '17']
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        #node: ['8', '10', '12', '14', '16', '17']
+        #os: [ubuntu-latest, windows-latest, macos-latest]
+        # let's make it a little bit simple for now
+        # current target version will be 14.
+        # TODO: investigate the multipe version matrix with single Vertica instance
+        node: ['14']
```

Comment:
We should probably log a jira to expand this once things settle down with testing

Reply:
I have created one for that